content/_code-samples/delete-account/js/delete-account.js

```diff
@@ -0,0 +1,65 @@
+if (typeof module !== "undefined") {
+    // Use var here because const/let are block-scoped to the if statement.
+    var xrpl = require('xrpl')
+  }
+  // Blackhole an account.
+  // For some addresses, it's possible that no one has the secret key, in which case the account is a black hole and the XRP is lost forever.
+  // https://xrpl.org/accounts.html#special-addresses
+  // https://xrpl.org/disable-master-key-pair.html#disable-master-key-pair
+
+  async function main() {
+    // Connect to a testnet node
+    console.log("Connecting to Testnet...")
+    const client = new xrpl.Client('wss://s.altnet.rippletest.net:51233')
+    await client.connect()
+  
+    // Get account credentials from the Testnet Faucet
+    console.log("Requesting addresses from the Testnet faucet...")
```

Comment:
nit: Only getting one account.
```suggestion
    console.log("Requesting an account from the Testnet faucet...")
```